start_pose_num: 88
end_pose_num: 88 
headYaw_min: -0.6
headYaw_max: 0.2
headYaw_step: 0.4
headPitch_min: -0.5
haedPitch_max: 0.2
headPitch_step: 0.33
image_topic: "/nao_camera/image_raw"